addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -43,6 +43,29 @@
                 <field name="employee_id"/>
                 <field name="holiday_status_id"/>
                 <field name="name"/>
+                <filter domain="[
+                        ('state','in',['confirm']),
+                        '|',
+                        ('employee_id.user_id', '!=', uid),
+                        '&amp;',
+                        ('employee_id.user_id', '=', uid),
+                        ('employee_id.leave_manager_id', '=', uid)]"
```

Comment:
?

Reply:
If you're set as your own manager to confirm leaves, you'll be able to see your own leaves when using the "waiting for me" filter. This ensures that if you're the only leave manager in the company, you can confirm your leaves without having trouble to find them